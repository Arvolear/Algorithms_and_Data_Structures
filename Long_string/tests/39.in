361132915961116940820479977120878191505093226020
14164251163