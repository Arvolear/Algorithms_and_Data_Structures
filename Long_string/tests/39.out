361132915961116940820479977120878191519257477183
361132915961116940820479977120878191490928974857
5115177324899831891795485690152612315607353447077206861260
25496082482953422394101327834585941358